src/libstd/error.rs

```diff
@@ -182,6 +182,13 @@ impl Error for string::FromUtf16Error {
     }
 }
 
+#[stable(feature = "str_parse_error2", since = "1.8.0")]
+impl Error for string::ParseError {
+    fn description(&self) -> &str {
+        "parse error"
```

Comment:
@alexcrichton: Done in https://github.com/dirk/rust/commit/365deeb19a2cff74e3d215622b1ab411581b99e9!